service/controller/app/v1/resource/chart/create.go

```diff
@@ -29,7 +29,7 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 			return microerror.Mask(err)
 		}
 
-		_, err = cc.G8sClient.ApplicationV1alpha1().Charts(cr.Namespace).Create(&chart)
+		_, err = cc.G8sClient.ApplicationV1alpha1().Charts(cr.ObjectMeta.Namespace).Create(&chart)
```

Comment:
The namespace will now be set in the desired state. So we use that here.

Reply:
I think you can also do `cr.GetNamespace()`.

Reply:
Good shout I went with that. The CR also has a Namespace field in the spec so I think calling GetNamespace is the clearest option.